portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches the data from our Java servlet 
+ */
+const getData = async () => {
```

Comment:
I'm curious, where did you learn this syntax? I haven't seen this yet :) Normally I see something like: `async function getData() {` (which would also match the other function definitions in the file)

Reply:
I just got used to that syntax since that's how I wrote some components with React. I also find it the most aesthetically good looking out of all the options :)